requirements.txt

```diff
@@ -1,4 +1,8 @@
-firebase-admin==5.0.1
-lark==1.1.2
-matplotlib==3.2.2
-numpy==1.19.0
+# firebase-admin==5.0.1
+firebase-admin
+# lark==1.1.7
+lark
+# matplotlib==3.2.2
+matplotlib
+# numpy==1.19.0
+numpy
```

Comment:
update to latest versions of dependent libs - this is likely debatable.

Reply:
Without pinning on the specific version, I am afraid that firebase-admin may break in the future. But I agree that we should try to push forward with the latest version of other modules.